code/src/App.js

```diff
@@ -1,9 +1,88 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
+
+
+import ThoughtForm from './components/ThoughtForm'
+import ThoughtList from './components/ThoughtList'
+
+import { API_URL, HEARTS_URL } from './reusable/Urls'
 
 export const App = () => {
+  const [thoughtList, setThoughtList] = useState([])
+  const [thoughtNew, setThoughtNew] = useState('')
+
+  useEffect(() => {
+    fetchThoughtList()
+  }, [])
+
+  //FETCH TO IMPORT THOUGHTS
+  const fetchThoughtList = () => {
+    fetch (API_URL)
+      .then(response => response.json())
+      .then(thought => setThoughtList(thought))
+      .catch(err => console.error(err))
```

Comment:
You shouldn't print anything to console in production code. Always remove functions like this after debugging.